core/examples/luigi-sample-angular/src/luigi-config/extended/navigation.js

```diff
@@ -272,8 +272,11 @@ class Navigation {
     return items;
   };
 
-  // The following configuration will be used to render a product switcher component
+  // The following configuration will be used to render a product switcher at the end of the top navigation
   productSwitcher = {
+    label: 'My Products',
```

Comment:
If there is a empty object `prodcutSwitcher={}` we get the following view on mobile.
![Bildschirmfoto 2019-07-31 um 16 58 57](https://user-images.githubusercontent.com/14272184/62223202-d45f2080-b3b4-11e9-9be9-0a97674ca21a.png)

Reply:
Thanks, I have added the default label _My Products_. Is also required for backwards compatibility.

Reply:
👍